docs/components/modeler/web-modeler/process-applications.md

```diff
@@ -83,11 +80,12 @@ will always use their latest state.
 
 After the process instance has been started, you will receive a notification with a link to the process instance view in
 [Operate](../../operate/operate-introduction.md).
-Follow this link to observe the progress of the process instance and interact with it if required.
+Follow this link to monitor the process instance and interact with it as needed.
 
 :::info
-If you click the **Deploy** or **Run** button on the modeling screen for a BPMN or DMN diagram that is part of a process
-application, you will be deploying all resources of the process application and not just the current diagram.
+Single-file deployment is not supported in a process application.
+If you click the **Deploy** or **Run** button on the modeling screen for any BPMN or DMN diagram in the process
+application, you will deploy _all_ resources of the process application and not just the current diagram.
```

Comment:
When the redirect we described earlier today is implemented, here's a good place to mention it

Reply:
Yes, I will update this page again once we have implemented the change.